tracing/span.go

```diff
@@ -78,16 +78,15 @@ func AddErrorCurrentSpan(ctx context.Context, err error) error {
 
 //AddErrorSpan adds error into span
 func AddErrorSpan(span *trace.Span, err error) error {
-	var code int32 = trace.StatusCodeUnknown
+	var code int32 = trace.StatusCodeOK
 	status, ok := status.FromError(err)
```

Comment:
The reuse of `status` here from module to variable makes the code harder to understand.